http/headers/Content-Security-Policy.json

```diff
@@ -391,6 +391,39 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "blocks_redirects": {
+            "__compat": {
+              "description": "Redirects are blocked after a form submission",
+              "support": {
+                "chrome": {
+                  "version_added": "≤63"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": null
```

Comment:
Safari also blocks per this 2018 comment https://github.com/w3c/webappsec-csp/issues/8#issuecomment-406463864